site/content/docs/5.2/forms/layout.md

```diff
@@ -174,6 +188,13 @@ At times, you maybe need to use margin or padding utilities to create that perfe
 
 Be sure to use `.col-form-label-lg` to your `<label>`s or `<legend>`s to correctly follow the size of `.form-control-lg`.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This form variant can not be used because it does not respect the Orange Design System specifications. The form text input label should always be placed above the input field.
```

Comment:
```suggestion
This form variant can not be used because it does not respect the Orange Design System specifications. The input label should always be placed above the input field.
```

Reply:
OK

Reply:
OK for LMP modification